requirements.txt

```diff
@@ -1,6 +1,7 @@
 adal==0.4.3
 applicationinsights==0.10.0
 argcomplete==1.8.0
+azure_devtools==0.1.0
```

Comment:
The published package name uses a hyphen: https://pypi.python.org/pypi/azure-devtools

The underscore was erroneous, and is in any case now removed entirely from `requirements.txt` per @troydai's comment.